Sources/FINN/FilterMarkets/FilterMarketB2B.swift

```diff
@@ -98,6 +98,10 @@ extension FilterMarketB2B: FilterConfiguration {
         }
     }
 
+    public var contextFilters: Set<FilterKey> {
```

Comment:
Not related to this PR, but it's weird that FINN-specific `FilterKey` type is a part of `FilterConfiguration` protocol for general use.

Reply:
I think this configuration part needs an update at some point.